core/misc/collapse.js

```diff
@@ -71,7 +71,7 @@ Backdrop.behaviors.collapse = {
       $fieldset.
         bind('summaryUpdated', function () {
           var text = $.trim($fieldset.backdropGetSummary());
-          summary.html(text ? ' (' + text + ')' : '');
+          summary.html(text ? text : '');
```

Comment:
Wondering if this could become:

```suggestion
          summary.html($.trim($fieldset.backdropGetSummary()));
```

Does that work in JS? ...don't know

Reply:
Yes, I think that would have been fine as well. I have no problem with the current syntax either though, so I've merged this as-is.